lib/chef_zero/endpoints/organization_policies_endpoint.rb

```diff
@@ -0,0 +1,99 @@
+
+module ChefZero
+  module Endpoints
+    # /organizations/NAME/policies
+    class OrganizationPoliciesEndpoint < RestBase
+      def set_opts
+        [ :create_dir, :data_store_exceptions ]
```

Comment:
`:create_dir` creates any intermediate directories; `:data_store_exceptions` raises internal data store exceptions instead of RestExceptions (which bubble up to become proper user-facing errors), and I'll remove that.